.gitignore

```diff
@@ -21,7 +21,7 @@
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
+.vscode/settings.json
```

Comment:
question (non-blocking): any reason for this change?

Reply:
yes, it did not ignore `.vscode/settings.json`

Reply:
i understand what it does, just not sure about the reasons, that's all, not a vscode user myself, and looking at something seemingly random with ignore this but add that looks strange, and was curious. anyways :)